getting_started/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -223,11 +223,10 @@ Other differences:
 GDScript          C#
 ================  ==================================================================
 Color8            Color.Color8
-is_inf            float.IsInfinity
-is_nan            float.IsNaN
 dict2inst         ? TODO
 inst2dict         ? TODO
 load              GD.load which is the same as ResourceLoader.load
+yield             await ToSignal(object, signal)
```

Comment:
The left column should be "yield on signal", since yield can also be used in GDScript to simply suspend a function to manually resume it.
I would add some description to the right part instead of just a code example. At least, to make it clear that it's not exactly the same as yield on signal in the way they work internally.

Reply:
What description ? Say - I will add